### ISSUE TITLE
Interface with new stepper motor

### ISSUE THREAD

**Issue by AbBaSaMo** [Monash-Connected-Autonomous-Vehicle/ESDA#42]
While we're waiting on the outcome for issue

**Comment by Jiawei-Liao** [comment]
Do you know what the servo brand/model is?

**Comment by AbBaSaMo** [comment]
@Jiawei-Liao not really. the brand and serial number is on the image someone on slack messaged under the software channel. You'll have to ask them or maybe @shrayb would know
